src/pyocd-server.ts

```diff
@@ -63,6 +63,13 @@ export class PyocdServer extends AbstractServer {
         ];
     }
 
+    protected resolveServerEnv(): NodeJS.ProcessEnv {
+        // Remove env variables and reset PATH
+        return {
+            PATH: ''
```

Comment:
Are we sure there are no other env variables needed? I can't think of any.

Reply:
I'll check pyOCD code for its use of env variables

Reply:
There are a few:
Like:
- `TOX_ENV_NAME`
- `PYOCD_HISTORY`
- `PYOCD_HISTORY_LENGTH`

We do not use any of them at the moment. It doesn't mean however that others are doing the same. I'll change it to pass `gdbServerEnv` from the client instead. That way each client using `cmsis-debug-adapter` will be able to decide.